app/controllers/organizations/staff/external_form_upload_controller.rb

```diff
@@ -7,6 +7,17 @@ def index
         authorize! :external_form_upload,
           context: {organization: Current.organization}
       end
+
+      def create
+        authorize! :external_form_upload,
+          context: {organization: Current.organization}
+        files = params.require(:files)
+        files.each do |file|
```

Comment:
I think we should only really allow 1 upload here...it doesn't make sense for more than one file. 

Given we are using `app/views/organizations/staff/shared/_attachment_form.html.erb` partial, we could make the value for the `multiple` attribute an argument that gets passed to the partial. So in this case, it would be `false` and in the other place the partial is used, on the pet, it would be `true`. Thoughts?

Then we can get rid of this loop here. 

Also, curious of the benefit of checking `file.is_a?(ActionDispatch::Http::UploadedFile)`

Reply:
`file.is_a?(ActionDispatch::Http::UploadedFile)` It is because the received parameter contains other things like `""`. Maybe It will be unnecessary when using just one file.

I don't understand completely the use case due to a lack of background knowledge of the project. If you think a single file is the way to go, I will implement the suggested changes.

Reply:
Yes this import service is making the assumption there is only one third party form, and therefore one CSV to upload. It doesn’t currently handle the case where adopter data lives across multiple forms. So it would be good to only allow one file for now.